src/System.Management.Automation/ComInterop/ComHresults.cs

```diff
@@ -3,6 +3,10 @@
 
 namespace System.Management.Automation.ComInterop
 {
+    /// <remarks>
+    /// This class is derived from
+    /// <see href="https://github.com/dotnet/runtime/blob/1f688ed75133664aa20c838fa451e2ecf9f1f424/src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/ComInterop/ComHresults.cs">dotnet/runtime</see>.
+    /// </remarks>
```

Comment:
There is no need to add this remark. The code in `ComInterop` folder are all ported from .NET.

Reply:
Although we have `src/System.Management.Automation/engine/ComInterop/README.md`, this PR moves `ComHresults.cs` into a new `ComInterop` directory in `src/System.Management.Automation`. 

I also think documenting the provenance to a specific commit is worthwhile.

Reply:
Ah, I didn't notice you moved the file. Please don't move the file.
We try to keep the `ComInterop` as intact as possible so in future if we need to port new changes to COM support from .NET, it's easier to do diff.

If moving the file is required for this cleanup to work, then I'd rather to continue to use `Utils.Succeeded`.